File: server/tests/fixtures/alpha_rule_triggers/csharp/naming_project_term_inconsistency.cs
// Should trigger: naming.project_term_inconsistency
// Rule detects inconsistent verb usage for same noun phrase (e.g., GetUser vs FetchUser)

public class UserService {
    public void GetUser(int id) {}  // Uses "Get" verb
    
    public void FetchUser(int id) {}  // Uses "Fetch" verb - inconsistent!
    
    public void LoadUser(int id) {}  // Uses "Load" verb - inconsistent!
}

class OrderManager {
    public void CreateOrder(object data) {}
    
    public void MakeOrder(object data) {}  // "Make" is synonym for "Create" - inconsistent!
}

File: server/tests/fixtures/alpha_rule_triggers/typescript/naming_project_term_inconsistency.ts
// Should trigger: naming.project_term_inconsistency
// Rule detects inconsistent verb usage for same noun phrase (e.g., getUser vs fetchUser)

class UserService {
    getUser(id: number): void {}  // Uses "get" verb
    
    fetchUser(id: number): void {}  // Uses "fetch" verb - inconsistent!
    
    loadUser(id: number): void {}  // Uses "load" verb - inconsistent!
}

class OrderManager {
    createOrder(data: any): void {}
    
    makeOrder(data: any): void {}  // "make" is synonym for "create" - inconsistent!
}
